cfme/base/__init__.py

```diff
@@ -223,7 +223,11 @@ def all(self):
         parent = self.filters.get('parent')
         for zone in zone_collection:
             zone.reload(attributes=['region_number'])
-            if parent and zone.region_number != parent.number:
+            # starting in 5.11 there's a maintenance zone that is not shown in the UI but
+            # is visible in API. We need to skip it for the correct navigation.
+            # https://bugzilla.redhat.com/show_bug.cgi?id=1455145
+            maintenance = 'Maintenance Zone'
```

Comment:
optional:
I would avoid catching this in a variable better to use directly.

Reply:
I was struggling how to format this :(